<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tela_cadJogo</class>
 <widget class="QMainWindow" name="Tela_cadJogo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tela Deletar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 158, 61)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deletar Jogo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>56</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jogo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delJogo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(142,142,142);
Color:rgb(0,0,0);
border: 2px solid #500;
 border-radius: 10px;
  border-style: outset;</string>
    </property>
    <property name="text">
     <string>Deletar</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sair">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(142,142,142);
border: 2px solid #500;
border-radius: 10px;
border-style: outset;</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>80</width>
      <height>28</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="deltodos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Deletar Todos</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
